### ISSUE TITLE
Make focus indicators in Desktop Modeler more obvious (a11y)

### ISSUE THREAD

**Issue by lmbateman** [camunda/camunda-modeler#4632]
### Problem you would like to solve

When tabbing through the Desktop Modeler header and footer, the current focus is not always visible, and even when it is, it is often too subtle.

### Proposed solution

- The focused element should always be indicated.
- The focus indicator should be obvious (e.g., by meeting contrast guidelines).

This may require some visual design work.

### Alternatives considered

Leave the focus indicators as they are.

### Additional context

The focus is on Camunda 7 in the footer. If you're not following along with the tabs, it's easy to miss.
![image](https://github.com/user-attachments/assets/920aae50-b887-4d93-a2e0-f7d8ad6b345a)

**Comment by abdul99ahad** [comment]
I’ve added my comments to the [Figma designs](https://www.figma.com/design/H9FhE4WddX0DvKQcEyk28X/Desktop-Modeler-Focus-States?node-id=0-1&node-type=canvas&t=T8rKrZQNMuBeAfV1-0), reflecting the focus indicator improvements I implemented in the `focus-indicators` branch. 

cc: @lmbateman
